sirius/web/twitter.py

```diff
@@ -1,51 +1,75 @@
-import os
-import datetime
-import flask
 import logging
-from gevent import pool
+
+from requests_oauthlib import OAuth2Session
+import requests
+import flask
 import flask_login as login
-from flask_dance.consumer import oauth_authorized, oauth_error
-from flask_dance.contrib.twitter import make_twitter_blueprint, twitter
-import twitter as twitter_api
 
 from sirius.models import user as user_model
 from sirius.models.db import db
 
 logger = logging.getLogger(__name__)
 
-api_key=os.environ.get('TWITTER_CONSUMER_KEY', 'DdrpQ1uqKuQouwbCsC6OMA4oF')
-api_secret=os.environ.get('TWITTER_CONSUMER_SECRET', 'S8XGuhptJ8QIJVmSuIk7k8wv3ULUfMiCh9x1b19PmKSsBh1VDM')
+blueprint = flask.Blueprint("twitter", __name__)
+
+auth_url = "https://twitter.com/i/oauth2/authorize"
+token_url = "https://api.twitter.com/2/oauth2/token"
+
+scopes = ["tweet.read", "users.read", "offline.access"]
```

Comment:
Do we need `tweet,read`? I also wonder about `users.read`?
```suggestion
scopes = ["users.read", "offline.access"]
```

Reply:
These are the scopes listed under the docs for `GET /2/users/me`:

https://developer.twitter.com/en/docs/twitter-api/users/lookup/api-reference/get-users-me